server/service/osquery_utils/queries.go

```diff
@@ -344,6 +344,14 @@ var extraDetailQueries = map[string]DetailQuery{
 		DirectIngestFunc: directIngestChromeProfiles,
 		Discovery:        discoveryTable("google_chrome_profiles"),
 	},
+	"battery": {
+		Query:            `SELECT serial_number, cycle_count, health FROM battery;`,
+		Platforms:        []string{"darwin"},
+		DirectIngestFunc: directIngestBattery,
+		// the "battery" table doesn't need a Discovery query as it is an official
+		// osquery table on darwin (https://osquery.io/schema/5.3.0#battery), it is
+		// always present.
```

Comment:
This assumption is correct, right?

Reply:
I believe so, yes, no need for `discovery`.